src/index.js

```diff
@@ -19,21 +19,23 @@ export default function (options = {}, callback = function noop () {}) {
   let testAttempt = 1
   let parsedOptions = parseOptions(options)
   let parser = getParser(parsedOptions.parser)
+  let plainLogger = new Logger(null)
```

Comment:
We need a way to tell the `Logger()` to color or not to color a message. If we don't do this all the messages/output will be colored, even the ones generated by protractor because of:
https://github.com/NickTomlin/protractor-flake/blob/v2.2.0/src/index.js#L62
https://github.com/NickTomlin/protractor-flake/blob/v2.2.0/src/index.js#L68

What I want to achieve is only color the message that are flaky related. Like these:
https://github.com/NickTomlin/protractor-flake/pull/48/files#diff-1fdf421c05c1140f6d71444ea2b27638R35
https://github.com/NickTomlin/protractor-flake/pull/48/files#diff-1fdf421c05c1140f6d71444ea2b27638R37

How do you suggest I tell the Logger() when to color and when not to color a message?